packages/ra-keycloak/src/authProvider.ts

```diff
@@ -173,12 +176,8 @@ export const keycloakAuthProvider = (
     async handleCallback() {
         await initKeyCloakClient(keycloakClient, options.initOptions);
         await isAuthenticated(keycloakClient, options.authenticationTimeout);
-
         if (keycloakClient.authenticated && keycloakClient.token) {
-            return {
-                redirectTo:
-                    localStorage.getItem(PreviousLocationStorageKey) || '/',
-            };
+            return;
```

Comment:
Same, just a cleanup since the `useHandleAuthCallback` hook already does that for us.